sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -177,6 +214,10 @@ public Read from(String filepattern) {
       return from(ValueProvider.StaticValueProvider.of(filepattern));
     }
 
+    public Read withSplit() {
```

Comment:
Why do we need this option ? We should be able to safely always enable splitting for runners that support SDF without an explicit flag from the user.

Reply:
I think Splittable dofn is not available for all runners yet so maybe we should still keep both function.